terra/contracts/wormhole/src/contract.rs

```diff
@@ -48,6 +50,7 @@ pub fn init<S: Storage, A: Api, Q: Querier>(
         wrapped_asset_code_id: msg.wrapped_asset_code_id,
         owner: deps.api.canonical_address(&env.message.sender)?,
         is_active: true,
+        fee: Coin::new(0, ""),  // No fee by default
```

Comment:
It is just idea, not related with security or performance.

You can use Optional type and can put None or Some(Coin::new())

Reply:
Yes, I also thought about this, but chose 0-sum coin simply for more effective storage strategy (Option takes more space when serialized). But looking at it again I have to agree, using Option would be more elegant.